packages/kbn-config/src/types.ts

```diff
@@ -26,3 +27,5 @@ export interface EnvironmentMode {
   dev: boolean;
   prod: boolean;
 }
+
+export type BuildFlavor = 'serverless' | 'traditional';
```

Comment:
I went with `traditional` for the "unflavored" version, but I can go with something else if we have a better idea.